sql/core/src/test/resources/sql-tests/inputs/pipe-operators.sql

```diff
@@ -821,6 +819,84 @@ select 1 x, 2 y, 3 z
 table other
 |> aggregate b group by a;
 
+-- WINDOW operators (within SELECT): positive tests.
+---------------------------------------------------
+
+-- SELECT with a WINDOW clause.
+table windowTestData
+|> select cate, sum(val) over w
+   window w as (partition by cate order by val);
+
+-- SELECT with RANGE BETWEEN as part of the window definition.
+table windowTestData
+|> select cate, sum(val) over w
+   window w as (order by val_timestamp range between unbounded preceding and current row);
+
+-- SELECT with a WINDOW clause not being referred in the SELECT list.
+table windowTestData
+|> select cate, val
+    window w as (partition by cate order by val);
+
+-- multiple SELECT clauses, each with a WINDOW clause (with the same window definition names).
+table windowTestData
+|> select cate, val, sum(val) over w as sum_val
+   window w as (partition by cate)
+|> select cate, val, sum_val, first_value(cate) over w
```

Comment:
The `first_value` result is nondeterministic since the window `w` has no ordering property. Please add an `order by` clause inside the window definition and regenerate the test result to make the test cases deterministic.

Same for each of the other test cases below using `first_value` on a window definition with no ordering requirement.